routes/plannedRoutes.js

```diff
@@ -0,0 +1,38 @@
+var db = require("../models");
+
+module.exports = function(app) {
+  app.get("/api/plan/:userid/", function(req, res) {
+    db.Plan.findAll({
+      where: {
+        UserId: req.params.userid
+      }
+    }).then(function(dbPlan) {
+      res.json(dbPlan);
+    });
+  });
+
+  app.post("/api/plan", function(req, res) {
+    db.Plan.create(req.body).then(function(dbPlan) {
+      res.json(dbPlan);
+    });
+  });
+  app.delete("/api/plan/:id", function(req, res) {
+    db.Plan.destroy({
+      where: {
+        id: req.params.id
+      }
+    }).then(function(dbPlan) {
+      res.json(dbPlan);
+    });
+  });
+
+  app.put("/api/plan", function(req, res) {
+    db.Plan.update(req.body, {
```

Comment:
@clairestolp @jasmine-fitts @tjordanwells While doing the category/subCategory routes I found that if we send the whole req.body it updates the id as well. Not sure if this matters to everyone or not, but it made me go back and break it up to each part of the body I wanted updated. What does everyone think?